src/main/java/Chatbot/alain/Storage.java

```diff
@@ -0,0 +1,212 @@
+package chatbot.alain;
+
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Locale;
+import java.util.regex.Pattern;
+
+/**
+ * Handles the loading and saving of tasks, as well as processing user inputs for the Alain chatbot.
+ */
+public class Storage {
+    private String filePath;
+    private Ui ui = new Ui();
+    private Boolean alrBye;
```

Comment:
Perhaps a variable name that is more reflective of our Boolean naming standards could be used? eg isEnd